src/transformStateWithClones.js

```diff
@@ -65,7 +65,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const copy = { ...state };
```

Comment:
I think `copy` is not the best name, think about something more specific, like `copyOfState`, or `temporaryState`, or something else